README.template.md

```diff
@@ -59,39 +59,42 @@ There are a number of `docker-compose.yml` files provided by this repository:
 
 ## Override
 
-This repository ignores `docker-compose.override.yml` which will be included in
-any `docker compose` commands you invoke by default.
+This git repository does not track `docker-compose.override.yml` which will 
+be included by default in all `docker compose` commands you invoke.
 
 Two platform dependent templates that allow for access to the hosts `SSH agent`
 are provided for `development` environments.
 
-Simply copy the appropriate `docker-compose.PLATFORM.yml` file into
+Simply copy the appropriate* `docker-compose.PLATFORM.yml` file into
 `docker-compose.override.yml`, on your development machine.
 
 Any additional changes that are for your local / development environment can
-then be added to `docker-compose.override.yml`.
+then be added to `docker-compose.override.yml`. Because they're not tracked...
 
 ## Building
 
-You can build your locally using `docker compose`
+You can build your development environment locally using `docker compose`
 
 ```bash
 docker compose --profile dev build
 ```
+This has the effect of...
 
-## Pulling
+## Pulling Docker Images
 
-The `docker compose` file provided require that you first pull the islandora
+The `docker compose` file provided requires that you first pull the islandora
 images with the following command:
 
 ```bash
 docker compose --profile dev --profile prod pull --ignore-buildable --ignore-pull-failures
 ```
 
+Both `dev` and `prod` are included so that...
+
 ## Running / Stoping / Destroying
 
-You must specify a profile either `dev` or `prod` to use `docker compose` files
-provided by this repository.
+You must specify a profile - either `dev` or `prod` - to use the `docker compose` files
```

Comment:
`docker compose` is the name of the command, `docker-compose[-foo].yml` is the name of the files. I feel weird referring to the files by the command, but if that's common practice I'll let it be.